rholang/examples/tut-registry.rho

```diff
@@ -0,0 +1,106 @@
+new test1ID, test3ID, ri(`rho:registry:insertArbitrary`), 
+                      //rs(`rho:registry:insertSigned:ed25519`), 
+                      rl(`rho:registry:lookup`), 
+                      stdout(`rho:io:stdout`), 
+                      stdoutAck(`rho:io:stdoutAck`), ack in { 
+    
+    @"REGISTRY_SIMPLE_INSERT_TEST"!(*test1ID)
```

Comment:
Sending on the forgeable names soon will be a noop. Please use unforgeable ones otherwise these examples will become incorrect soon.

Reply:
Switched to using unforgeable names.